x/fundraising/keeper/grpc_query.go

```diff
@@ -126,7 +126,14 @@ func (k Querier) Bids(c context.Context, req *types.QueryBidsRequest) (*types.Qu
 	case req.Bidder == "" && req.Eligible != "":
 		bids, pageRes, err = queryBidsByEligible(ctx, k, store, req)
 	case req.Bidder != "" && req.Eligible != "":
-		bids, pageRes, err = queryBidsByBidder(ctx, k, store, req, true)
+		var eligible bool
+		if req.Eligible != "" {
```

Comment:
This indent looks already conditioned `req.Eligible != ""`, isn't it?

Reply:
Since `req.Eligible` is string type, I added a condition to check once again and parse it to bool type. However, as you said it will be cleaner to change eligible to string type in `queryBidsByBidder` and parse it to bool type inside the function. I'll go make that change.

Reply:
Misunderstood the question. As you explained offline, I will remove the duplicate condition check.